Configuration/Adaptive_Meshing.cfg

```diff
@@ -20,7 +20,9 @@ gcode:
     {% set detach_macro = kamp_settings.detach_macro | string %}                                                                    # Pull detach probe command from _KAMP_Settings
     {% set mesh_margin = kamp_settings.mesh_margin | float %}                                                                       # Pull mesh margin setting from _KAMP_Settings
     {% set fuzz_amount = kamp_settings.fuzz_amount | float %}                                                                       # Pull fuzz amount setting from _KAMP_Settings
+    {% set mesh_density = kamp_settings.mesh_density | float %}                                                                     # Pull mesh density setting from _KAMP_Settings
```

Comment:
```suggestion
    {% set mesh_density = kamp_settings.variable_mesh_density | float %}                                                                     # Pull mesh density setting from _KAMP_Settings
```

Reply:
I don’t think `variable` needs to be in there, it’s not required, and it also doesn’t match the other variables in the meshing macro. Appreciate the review though!

Reply:
I am not suggesting that the name of the variable is changed. The author has used different names in two places, so I was simply matching the other place. I agree with your statement though, so perhaps it is better to remove `variable_` from the other one instead of adding it here.

Reply:
Or maybe I am just confused about how variables work, actually. Feel free to ignore x)